envs/conda_build_config.yaml

```diff
@@ -168,6 +178,8 @@ openjdk:
   - 1.8.0                      # [ppc64le]
 openmpi:
   - 3.1.3
+optional_lite:
```

Comment:
ONNX sure loves their C++ extensions! For this one I don't really see any compatibility guidance.

Reply:
Thank you ! :)